nislmigrate/migrators/file_migrator.py

```diff
@@ -152,3 +160,42 @@ def pre_restore_check(
 
     def add_additional_arguments(self, argument_manager: ArgumentManager):
         argument_manager.add_switch(_METADATA_ONLY_ARGUMENT, help=_METADATA_ONLY_HELP)
+        argument_manager.add_argument(_CHANGE_FILE_STORE_ARGUMENT, help=_CHANGE_FILE_STORE_HELP, metavar='update')
+
+    def update_root_file_path_in_metadata(self, configuration: _FileMigratorConfiguration):
+        mongo_configuration = configuration.mongo_configuration
+        old_path = configuration.data_directory
+        new_path = configuration.update_store_path
+        path_field_name = 'path'
+        collection_name = self.name.lower()
+        does_path_field_start_with_old_path = self.does_field_start_with_prefix_predicate(path_field_name, old_path)
+        replace_old_path_with_new_path = self.replace_prefix_in_document_function(path_field_name, old_path, new_path)
+
+        configuration.mongo_facade.update_documents_in_collection(
+            mongo_configuration,
+            collection_name,
+            does_path_field_start_with_old_path,
+            replace_old_path_with_new_path)
+
+    @staticmethod
+    def does_field_start_with_prefix_predicate(field: str, prefix: str) -> Callable[[Dict[str, Any]], bool]:
+        return lambda document: document[field].startswith(prefix)
```

Comment:
This may be a bit more generic than you need here. You could just have a does_path_start_with_prefix_predicate and simplify.